app/controllers/admin/facility_groups_controller.rb

```diff
@@ -41,7 +41,7 @@ def update
   end
 
   def destroy
-    @facility_group.destroy
+    @facility_group.discard
```

Comment:
Should there be a check to see if the group is empty (has no facilities) ?

Reply:
@divs1210 Added destroyability checks for FG in the controller.